test/e2e/utils.go

```diff
@@ -0,0 +1,196 @@
+// +build e2e
```

Comment:
need to add copyright header

Reply:
@skriss commit `644695b` resolves this comment.